src/v2/Apps/Consign/Components/ContactUs.tsx

```diff
@@ -1,20 +1,58 @@
 import React from "react"
-import { Button, Text } from "@artsy/palette"
+import { Button, Flex, Link, Text } from "@artsy/palette"
 import { SectionContainer } from "./SectionContainer"
+import { Media } from "v2/Utils/Responsive"
 
-export const ContactUs: React.FC = () => {
+export const ContactUs: React.FC<{ darkVariant?: boolean }> = ({
+  darkVariant,
+}) => {
   return (
-    <SectionContainer background="black5">
-      <Text width="100%" textAlign="left" mb={3} variant="largeTitle">
-        Need to speak to a specialist? Contact us.
+    <SectionContainer background={darkVariant ? "black100" : "black5"}>
+      <Text
+        width="100%"
+        textAlign="left"
+        mb={3}
+        variant="largeTitle"
+        color={darkVariant ? "white100" : "black100"}
+      >
+        Questions? Speak to an Artsy Specialist
       </Text>
-
-      <Text variant="text" color="black60" mb={4}>
-        Schedule a call with one of our specialist team members for <br />
-        more information on how Artsy can sell your artwork.
-      </Text>
-
-      <Button variant="secondaryOutline">Book an appointment</Button>
+      <Flex flexDirection="row" flexWrap="wrap" mb={3}>
+        <Text
+          variant="text"
+          color={darkVariant ? "black5" : "black60"}
+          mr={0.5}
+        >
+          Email us at
+        </Text>
+        <Text
+          variant="mediumText"
+          color={darkVariant ? "black5" : "black60"}
+          mr={0.5}
+        >
+          consign@artsy.net
+        </Text>
+        <Text
+          variant="text"
+          color={darkVariant ? "black5" : "black60"}
+          mr={0.5}
+        >
+          or call
+        </Text>
+        <Text variant="mediumText" color={darkVariant ? "black5" : "black60"}>
```

Comment:
Was gonna say it might be worth it to store this in a var, but looking around, there's enough variation to where i think its actually clearer this way 👍